.github/pull_request_template.md

```diff
@@ -0,0 +1,34 @@
+## Checklist Before A PR is Raised
+Before raising a PR, you need to check and see if your PR:
+
+- [ ] Meets issue expectations for functionality and implementation (Read the issue cards properly)
+
+- [ ] Has the appropriate test coverage [Reference](https://github.com/gobble/guides/tree/master/code_review#test-quality)
+
+- [ ] Has no trailing whitespaces or typos. You can check out [styles guide](https://github.com/gobble/guides/tree/master/style)
```

Comment:
Perhaps since these guides are private we could exclude the links for now?

It would be great to have a public facing guide avail to users for Intelligent Foods as well.